doc/user/content/release-notes.md

```diff
@@ -50,6 +50,8 @@ Wrap your release notes at the 80 character mark.
 
 - Fix a bug that caused a panic when computing the `max` of `int2` values.
 
+- Support `ORDER BY` in aggregate functions.
```

Comment:
Is there a syntax diagram somewhere (select.md?) that needs to get updated?

Reply:
I didn't find an obvious place to add this. Postgres has a dedicated page for aggregate functions where this is discussed. We could add it there. Opened #8189 to track.